.github/workflows/deploy-happy-stack.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
         if: failure() && github.ref == 'refs/heads/main'
 
-  e2e-test:
+  e2e-logged-in-test:
```

Comment:
Run all logged in e2e tests in its own step, outside of the rest of the sharded e2e tests. In the future, if we have more logged in e2e tests, we can shard this step too